### ISSUE TITLE
Uploaded Files Not Visible

### ISSUE THREAD

**Issue by tdiprima** [open-webui/open-webui#6565]
**Description:**

After following the [Open WebUI documentation for uploading files](https://docs.openwebui.com/api/#uploading-files), I successfully uploaded several files using the suggested code snippet. However, these files are not visible in the interface when typing `#` in the prompt field. Based on previous functionality, I expected to see all uploaded documents prefixed with `#` for easy selection and access.

**Code Used for Uploading Files:**

```python
import requests

def upload_file(token, file_path):
    url = 'http://localhost:3000/api/v1/files/'
    headers = {
        'Authorization': f'Bearer {token}',
        'Accept': 'application/json'
    }
    files = {'file': open(file_path, 'rb')}
    response = requests.post(url, headers=headers, files=files)
    return response.json()
```

**Steps to Reproduce:**

1. Use the code above to upload a file to Open WebUI.
2. In the prompt field, type `#` to bring up the list of documents.
3. Observe that the uploaded files are not listed.

**Expected Behavior:**

All uploaded files should be visible in the interface, prefixed by `#`, to enable quick access and reference.

**Current Behavior:**

Uploaded files do not appear in the document list, despite the upload operation returning success.

**Environment:**

- **Open WebUI Version:** v0.3.35
- **OS:**macOS 14.1

**Additional Context:**

This affects workflows that rely on the ability to quickly view and reference uploaded documents through the prompt interface. Any guidance on resolving this or confirmation of a potential bug would be greatly appreciated.

---

Thank you for taking a look!

**Comment by medrpa** [comment]
Can you please tell me exactly those pdfs are scanned or searchable/digitize?

**Comment by tdiprima** [comment]
They are text documents.  So yes, they are searchable.

**Comment by medrpa** [comment]
In the knowledge base, we upload document files or directories into a collection. We then use the "#" symbol in the prompt field to select the appropriate collection.

**Comment by tdiprima** [comment]
> In the knowledge base, we upload document files or directories into a collection.

Right.  But I said I uploaded them via the script that was recommended in the [documentation](https://docs.openwebui.com/api/#uploading-files).  I haven't gotten to the [Adding Files to Knowledge Collections](https://docs.openwebui.com/api/#adding-files-to-knowledge-collections) part.  But I have questions about that code also: #6553 .

**Comment by tdiprima** [comment]
@medrpa could you please maybe tag somebody who is familiar with the REST API, and bring them into the conversation?
